CHANGELOG.md

```diff
@@ -304,7 +306,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Initial public release.
 
-[Unreleased]: https://github.com/slackhq/nebula/compare/v1.4.0...HEAD
+[Unreleased]: https://github.com/slackhq/nebula/compare/v1.5.0...HEAD
+[1.4.0]: https://github.com/slackhq/nebula/releases/tag/v1.5.0
```

Comment:
1.5.0?

Reply:
haha pushed the fix the same time you commented :)

Reply:
I see :smile: